configuration/parse.go

```diff
@@ -19,6 +22,9 @@ type Configuration struct {
 	Path    string
 }
 
+var unixComments = regexp.MustCompile(`#.*$`)
+var untilComment = regexp.MustCompile(`(until=)(.*)`)
```

Comment:
Question: Should these be a const instead of a var?

Reply:
See comment above about const